sources/packages/backend/apps/queue-consumers/src/main.ts

```diff
@@ -0,0 +1,47 @@
+require("../../../env_setup_apps");
+import { Queues } from "@sims/services/queue";
+import { ConfigService } from "@sims/utilities/config";
+import { createBullBoard } from "@bull-board/api";
+import { BullAdapter } from "@bull-board/api/bullAdapter";
+import { ExpressAdapter } from "@bull-board/express";
+import { NestFactory } from "@nestjs/core";
+import { Queue } from "bull";
+import { QueueConsumersModule } from "./queue-consumers.module";
+import * as basicAuth from "express-basic-auth";
+
+async function bootstrap() {
+  const app = await NestFactory.create(QueueConsumersModule);
+
+  const config = app.get<ConfigService>(ConfigService);
+
+  //Application port.
+  const port = config.queueConsumersPort;
```

Comment:
There is no need to declare a variable to store the property here.
It can be directly consumed as `await app.listen(config.queueConsumersPort)`.